cumulusci/core/config/project_config.py

```diff
@@ -266,7 +279,7 @@ def repo_name(self):
             return
 
         in_remote_origin = False
-        with open(os.path.join(self.repo_root, ".git", "config"), "r") as f:
+        with open(self.git_path("config"), "r") as f:
```

Comment:
The logic below for reading the file to find the "origin" url is duplicated between several of these functions. I'm thinking we should move all of it (except for the final step of parsing the url) into a helper function instead of just `git_path`.